src/lib/selection/media/html-selector.js

```diff
@@ -102,8 +113,14 @@ export default class HTMLSelector extends MediaSelector {
       range.setStart(start.offsetNode, start.offset)
       range.setEnd(end.offsetNode, end.offset)
     } else if (typeof doc.caretRangeFromPoint === 'function') {
-      start = doc.caretRangeFromPoint(startX, startY)
-      end = doc.caretRangeFromPoint(endX, endY)
+      if (eventType === 'MouseMove') {
+        start = doc.caretRangeFromPoint(startX * 0.95, startY)
```

Comment:
I don't fully understand the rationale behind the multipliers so maybe it would be good to have some comments, and also I think possibly make these values configurable?

Reply:
Added comment and made them - MouseMove settings